modules/Material/ApplicationWindow.qml

```diff
@@ -111,12 +111,39 @@ Controls.ApplicationWindow {
         z: 100
 
         property Item currentOverlay
+        property bool backdrop: false
+
+        Rectangle {
```

Comment:
How about instead of having the child rectangle, we make `overlayLayer` be a Rectangle, and set the color to something like `currentOverlay.overlayColor`? This would allow for future components requiring move background colors in the future.